fsd_utils/locale_selector/set_lang.py

```diff
@@ -33,10 +35,12 @@ def __init__(self, app):
 
     @staticmethod
     def select_language(locale):
-
-        response = make_response(
-            redirect(request.referrer or request.args.get("return_url") or "/", 302)
```

Comment:
It seems that we first check do we have request.referrer then if yes it redirects with status 302
& if we got request.args.get("return_url") then redirects to that finally if nothing it will go to /

Before redirection don't we need to do any url safe checks ?